### ISSUE TITLE
Implement ajax for domain zone editting

### ISSUE THREAD

**Issue by elguitar** [Seravo/seravo-plugin#182]
This is a subtask #12.

This should be done before making the frontend. It should be kind of a MVP of the final product which @nasusa can finnish after :) 

I will assign myself here.

**Comment by elguitar** [comment]
Hmmmm... Right now the implementation is using GET and POST requests, which works fine. Using AJAX would be cool and all but I don't think that it is a priority.
Does @JoosuaKoskinen have some input on this?

**Comment by JoosuaKoskinen** [comment]
Not really. It sure would be nice to have. The URL can get quite ugly with the modification responds etc.

eg. wp-admin/tools.php?page=domains_page&domain=pisara.eu&paged=1&zone-updated=true&modifications%5B0%5D=Changed+the+origin+A+and+AAAA+records+to+match+the+servers+addresses

**Comment by l3ku** [comment]
What about just making the GET/POST requests using `jQuery.get()` and `jQuery.post()` and dynamically modifying the DOM content based on the results and populating the data to the correct locations by CSS selectors? Of course this would require coding some JavaScript... :)

**Comment by Mainerii** [comment]
Should just the zone editing use AJAX or the whole domains-page (eg. records-table on 'view' link click)? Fetching the records with AJAX should be implemented anyways so the zone editing textarea can be updated to sanitized version after each zone update.

**Comment by elguitar** [comment]
Closed via 627b5068dd1727c4f1155694437bfbc51fc878ba
